system/Drivers/CMSIS/Device/ST/STM32YYxx_CMSIS_version.md

```diff
@@ -5,13 +5,8 @@
   * STM32F2: 2.2.5
   * STM32F3: 2.3.6
   * STM32F4: 2.6.8
-<<<<<<< HEAD
   * STM32F7: 1.2.7
```

Comment:
New CMCIS version for F7 is 1.2.8

Reply:
> New CMCIS version for F7 is 1.2.8

Nice catch @ABOSTM 
I will update it. I missed that when reviewed before the merge.
Looking deeply, it was weird as a merge conflict has been committed then remove the next commit. Really bad git usage method ended to an error in the file...

https://github.com/stm32duino/Arduino_Core_STM32/pull/1761/commits/7ea25271b0b516f85adb3c5c2e507a9ee343a641#diff-c2e123dabb25a1e1586c28bbe165ce41276ae6a37ac5445e5eca80300eb58297R8-R14

https://github.com/stm32duino/Arduino_Core_STM32/pull/1761/commits/91789d53bcee9c29a20a08bda9fcf9aa38a48ecc#diff-c2e123dabb25a1e1586c28bbe165ce41276ae6a37ac5445e5eca80300eb58297L8-L14

Reply:
fixed with https://github.com/stm32duino/Arduino_Core_STM32/commit/90b3c4fe0e8d6f701a8ea324f923c9a34247d6db